Spigot-Server-Patches/0261-PreCreatureSpawnEvent.patch

```diff
@@ -44,6 +44,7 @@ index 87fe4775f..9466bcdc7 100644
 +                                org.bukkit.event.entity.CreatureSpawnEvent.SpawnReason.SPAWNER
 +                        );
 +                        if (!event.callEvent()) {
```

Comment:
Make this configurable and default to false instead. then it wont be a behavior change.

Reply:
Should it really not set the delay of the spawner by default? I feel that's what one would expect when cancelling/aborting the event.

I mean I'm open to making it configurable but I doubt anyone is even using this part of the behaviour to warrant it to be disabled by default, and if someone really is they can just disable this delay?

Reply:
Hmm, on 2nd thought, maybe it's fine as is and doesn't need to be configurable, as a config option doesn't make sense in the context of not spawner spawns.

Nevermind on the change request.